src/openrct2-ui/interface/Window.h

```diff
@@ -26,6 +26,13 @@ struct Window : rct_window
     void SetCheckboxValue(rct_widgetindex widgetIndex, bool value);
     void DrawWidgets(rct_drawpixelinfo& dpi);
     void Close();
+    void TextInputOpen(rct_widgetindex call_widget,
+                             rct_string_id title,
+                             rct_string_id description,
+                             const Formatter& descriptionArgs,
+                             rct_string_id existing_text,
```

Comment:
still have snake case for these two parameters.

Reply:
Haha, that is my bad. Updated that right now